Purchases/Public/EntitlementInfo.swift

```diff
@@ -355,3 +296,27 @@ private extension EntitlementInfo {
     }
 
 }
+
+/**
+ This extension contains some internal helper structs to decode the data received from the backend.
+ */
+extension EntitlementInfo {
+
+    struct EntitlementData: Decodable {
+        let expiresDate: Date?
```

Comment:
Style: new lines for struct definition after `{` and before `}`

Reply:
👍🏼